lib/resty/websocket/client.lua

```diff
@@ -140,12 +141,11 @@ function _M.connect(self, uri, opts)
                    "client_priv_key must be provided with client_cert")
         end
 
-        if opts.ssl_verify or opts.server_name then
-            if not ssl_support then
-                return nil, "ngx_lua 0.9.11+ required for SSL sockets"
-            end
```

Comment:
note: this `ssl_support` check was superfluous, because it is checked [here](https://github.com/openresty/lua-resty-websocket/blob/fbd01942507b1cfa0cbbaa792c3ccd46a3d66316/lib/resty/websocket/client.lua#L176-L178).

#74 also moves the `ssl_support` check up earlier within the `connect()` method